src/data/polynomial/coeff.lean

```diff
@@ -285,4 +285,7 @@ end
 
 end cast
 
+instance polynomial.char_zero [char_zero R] : char_zero R[X] :=
```

Comment:
```suggestion
instance char_zero [char_zero R] : char_zero R[X] :=
```
Remove duplicate namespace.  Why did the linter not pick this up?

Reply:
Probably because it is an instance. I think there is no need to give it a name.

Reply:
You are right!  I removed the name, which coincides with the auto-generated one.